### PR TITLE
Add to collection modal

### PR DESCRIPTION
**Pull request by mirjamverloop**

### CODE REVIEW

File: components/user/ModalCollection.vue [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+          <b-button
+            variant="outline-primary"
+            @click="goBack()"
+          >
+            Go back
+          </b-button>
+          <b-button
+            variant="primary"
+            type="submit"
+          >
```
Comment (annachristaki) [inline components/user/ModalCollection.vue:39]: Shouldn't this be disabled before mandatory fields are entered?

File: assets/scss/modal.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        margin-bottom: 0;
+      }
+
+      .form-control {
+        background: $white;
+        border: 1px solid $paper;
+        font-size: $font-size-small;
+      }
+
+      .form-group:focus-within {
```
Comment (annachristaki) [inline assets/scss/modal.scss:17]: 👍

File: components/user/ModalCollection.vue [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+      <b-form @submit.stop.prevent="submitForm">
+        <b-form-group
+          label="Collection name"
+          label-for="collection"
+        >
+          <b-form-input
+            id="collection"
+            type="text"
+            required
+          />
```
Comment (spyrosbek) [inline components/user/ModalCollection.vue:19]: Shouldn't the form input demand the use of a validation method, to ensure the ticket's acceptance criteria (The title should have a maximum length of [whatever our card title max-length is])? As well as the one about the description input (The description should have a maximum length of 240 characters.).

File: components/user/ModalCollection.vue (hunk 1 above)
Comment (mirjamverloop) [reply components/user/ModalCollection.vue]: That is a good idea, but not clearly written in the acceptance criteria.

File: components/user/ModalCollection.vue (hunk 3 above)
Comment (mirjamverloop) [reply components/user/ModalCollection.vue]: Yes, thanks!

**Pull request merged**